spec/scrabble_spec.rb

```diff
@@ -0,0 +1,72 @@
+require './lib/scrabble.rb'
+
+describe Scrabble do
+
+  context "Scoring words" do
+
```

Comment:
There are all error scenarios so I may consider using a different context that is just for scoring words with errors.